doc/operators.sgm

```diff
@@ -393,7 +393,7 @@
                 The center operator <literal>@@</literal> is a non-boolean
                 unary operator returning the center of an object.  In the
                 current implementation of <application>pgSphere</application>,
-                only centers of circles and ellipses are supported. Instead
+                only centers of circles, ellipses, point, array of points are supported. Instead
```

Comment:
I think it should be `points, arrays of points` to be grammatically correct here. I think this line needs is too long, and it needs to be word-wrapped after `are`, along with the next couple of lines where needed.

Reply:
resolved